ionoscloud/test_constants.go

```diff
@@ -1896,6 +1901,9 @@ resource` + ` ` + constant.WireGuardGatewayResource + ` ` + constant.WireGuardGa
     day_of_the_week       = "Monday"
     time                  = "09:00:00"
   }
+  backup {
+	loc = "de"
```

Comment:
would be best to use clusterBackupLocationValue if you defined it?

Reply:
I removed that part entirely as resource_vpn_wireguard_gateway does not have a backup block